src/dependency.rs

```diff
@@ -105,6 +115,14 @@ impl Dependency {
         }
     }
 
+    /// Get the alias for the dependency (if any)
+    pub fn rename(&self) -> Option<&str> {
+        match &self.rename {
+            Some(rename) => Some(&rename),
+            None => None,
+        }
+    }
+
     /// Convert dependency to TOML
     ///
     /// Returns a tuple with the dependency's name and either the version as a `String`
```

Comment:
this method should be adjusted for the `cargo add` rename support in the future

Reply:
(this was supposed to be a comment on `to_toml` method on `Dependency`, but github didn't allow to comment below this line)

I was suggesting to change the `fn to_toml` to take `rename` into account. So if a dependency has a rename, it's toml representation should be e.g.
`te = { package = "toml_edit", version = "0.1" }`
instead of
`toml_edit = "0.1"`